src/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -79,6 +87,14 @@ private static bool TryReadMemoryValueFromFile(string path, out ulong result)
                         result = checked(ulongValue * multiplier);
                         return true;
                     }
+
+                    if (MaxBytes == bytes.AsSpan())
+                    {
+                        // This is not (UIn64.MaxValue, true) because we want to let the caller
+                        // fallback to reading the real limits via other means
+                        result = 0;
+                        return false;
+                    }
```

Comment:
Per my previous comments, I would just delete this whole if block and let it fall through to the `result = 0; return false;` below.  If you want to add a comment about "max", that's fine, but this code is just overhead for no actual benefit.

Reply:
Okay. I added a comment and removed the entire check.